JSQMessagesViewController/Model/JSQAudioMediaItem.m

```diff
@@ -240,10 +240,18 @@ - (UIView *)mediaView
         playView.clipsToBounds = YES;
 
         // create the play button
-        CGRect buttonFrame = CGRectMake(self.audioViewAttributes.controlInsets.left,
-                                        self.audioViewAttributes.controlInsets.top,
-                                        self.audioViewAttributes.playButtonImage.size.width,
-                                        self.audioViewAttributes.playButtonImage.size.height);
+        CGRect buttonFrame;
+        if (!self.appliesMediaViewMaskAsOutgoing) {
+            buttonFrame = CGRectMake(self.audioViewAttributes.controlInsets.left + 5,
```

Comment:
we don't want this hardcoded `5` here.

we should handle this somehow in `controlInsets`